airflow_clickhouse_plugin/operators/clickhouse_operator.py

```diff
@@ -18,6 +18,9 @@ def __init__(
             clickhouse_conn_id: str = default_conn_name,
             parameters: Optional[Dict[str, Any]] = None,
             database: Optional[str] = None,
+            with_column_types: Optional[bool] = False,
+            types_check: Optional[bool] = False,
+            query_id: Optional[str] = None,
```

Comment:
@1ng4lipt my idea for this PR was to add all the parameters supported by `execute` method to all three: hook, operator, and sensor. The execute parameters are available here: https://clickhouse-driver.readthedocs.io/en/latest/api.html#clickhouse_driver.Client.execute — could you please add them all and propagate through all the classes and the tests?

Could you please also check which _earliest_ version of `clickhouse-driver` supports all the current `execute` method arguments?

If you are busy to add the requested, please let me know: it's ok to accept the PR at the current stage. I will just create an issue to backlog adding other the parameters. Your contribution is already valuable, thanks! 🤝

Reply:
@bryzgaloff I see.. Could you please create an issue for all of this so I can modify those files later?